package.json

```diff
@@ -1,23 +1,14 @@
 {
-  "name": "ipfs-deploy",
-  "version": "12.0.0",
-  "description": "Zero-Config CLI to Deploy Static Websites to IPFS",
+  "name": "@thebuidler/ipfs-deploy",
+  "version": "1.0.0",
+  "description": "[Modified] Zero-Config CLI to Deploy Static Websites to IPFS",
```

Comment:
Why are you changing the original package's info to your own?

Reply:
Not meant to be merged, apologies for the noise. Testing exposing ipfs nodes.